import-automation/executor/main.py

```diff
@@ -49,14 +50,26 @@
 AUTO_IMPORT_JOB_STATUS_LOG_TYPE = "auto-import-job-status"
 
 
-def scheduled_updates(absolute_import_name: str, import_config: str):
+def _override_configs(absolute_import_name: str,
+                      config: configs.ExecutorConfig) -> configs.ExecutorConfig:
+    import_dir, import_name = absolute_import_name.split(':')
```

Comment:
`split(':', 1)` to limit to 2 parts in case there are more ':' chars. o/w will throw a ValueError exception.